Content.Shared/Mobs/Systems/MobThresholdSystem.cs

```diff
@@ -246,6 +245,10 @@ public bool GetScaledDamage(EntityUid target1, EntityUid target2, out DamageSpec
             return;
 
         CheckThresholds(target, mobState, threshold, damageable);
+
+        var ev = new MobThresholdChecked(target, mobState, threshold, damageable);
+        RaiseLocalEvent(target, ref ev, true);
+        UpdateAlerts(target, mobState.CurrentState, threshold, damageable);
```

Comment:
Does the alert update stuff fix https://discord.com/channels/310555209753690112/1063661223134101534 ?
or is that some unrelated bug? (I never read the refactor PR so I have NFI what half of this code does).

Reply:
The old mobstate system was hardcoded to avoid using that particular sprite so when I updated it to use the YAML for it's sprite states it started to show the wrong sprite. I've fixed that with a proper fix that allows the sprites to be fully configured from YAML